tests/Neo.Network.RPC.Tests/UT_WalletAPI.cs

```diff
@@ -127,6 +127,21 @@ public async Task TestTransferfromMultiSigAccount()
 
             var tranaction = await walletAPI.TransferAsync(NativeContract.GAS.Hash, 1, new[] { keyPair1.PublicKey }, new[] { keyPair1 }, UInt160.Zero, NativeContract.GAS.Factor * 100);
             Assert.AreEqual(testScript.ToHexString(), tranaction.Script.ToHexString());
+
+            try
+            {
+                tranaction = await walletAPI.TransferAsync(NativeContract.GAS.Hash, 2, new[] { keyPair1.PublicKey }, new[] { keyPair1 }, UInt160.Zero, NativeContract.GAS.Factor * 100);
```

Comment:
`transaction` this test was wrong designed because it has two possible paths. Pass, and Fail.